prov/verbs/src/fi_verbs.c

```diff
@@ -1990,7 +1990,7 @@ fi_ibv_eq_read(struct fid_eq *eq, uint32_t *event,
 
 	ret = rdma_get_cm_event(_eq->channel, &cma_event);
 	if (ret)
-		return (errno == EAGAIN) ? 0 : -errno;
+		return (errno == EAGAIN) ? -FI_EAGAIN : -errno;
```

Comment:
The FI error codes map to standard errno values where they exist.  The code is already taking advantage of this by returning -errno anyway.  We just don't want to limit ourselves to errno values only.  The FI_* definitions also provide some future flexibility for portability.